tests/cephfs/cephfs_mirroring/test_cephfs_mirroring_configure_cephfs_mirroring_ha.py

```diff
@@ -0,0 +1,343 @@
+
+        log.info(
+            "Fetch the daemon_name, fsid, asok_file, filesystem_id and peer_id to validate the synchronisation"
+        )
+        fsid = fs_mirroring_utils.get_fsid(cephfs_mirror_node[0])
+        log.info(f"fsid on ceph cluster : {fsid}")
+        daemon_name = fs_mirroring_utils.get_daemon_name(source_clients[0])
+        log.info(f"Name of the cephfs-mirror daemon : {daemon_name}")
+        asok_file1 = fs_mirroring_utils.get_asok_file(
+            cephfs_mirror_node[0], fsid, daemon_name[:-1]
+        )
+        asok_file2 = fs_mirroring_utils.get_asok_file(
+            cephfs_mirror_node[1], fsid, daemon_name[1:]
+        )
+        log.info(
+            f"Admin Socket file of {cephfs_mirror_node[0].node.hostname} daemon : {asok_file1}"
+        )
+        log.info(
+            f"Admin Socket file of {cephfs_mirror_node[1].node.hostname} daemon : {asok_file2}"
+        )
+        filesystem_id = fs_mirroring_utils.get_filesystem_id_by_name(
+            source_clients[0], source_fs
+        )
+        log.info(f"filesystem id of {source_fs} is : {filesystem_id}")
+        peer_uuid = fs_mirroring_utils.get_peer_uuid_by_name(
+            source_clients[0], source_fs
+        )
+        log.info(f"peer uuid of {source_fs} is : {peer_uuid}")
+
+        log.info("Validate the Snapshot Synchronisation on Target Cluster")
+        validate_sync_dir_on_mirror_node1 = fs_mirroring_utils.get_synced_dir_count(
+            cephfs_mirror_node[0], source_clients[0], source_fs, daemon_name[:-1]
+        )
+        log.info(
+            f"Total Synced directory found on {cephfs_mirror_node[0].node.hostname} "
+            f"is {validate_sync_dir_on_mirror_node1}"
+        )
+
+        validate_sync_dir_on_mirror_node2 = fs_mirroring_utils.get_synced_dir_count(
+            cephfs_mirror_node[1], source_clients[0], source_fs, daemon_name[1:]
+        )
+        log.info(
+            f"Total Synced directory found on {cephfs_mirror_node[1].node.hostname} "
+            f"is {validate_sync_dir_on_mirror_node2}"
+        )
+
+        total_dir_synced = int(validate_sync_dir_on_mirror_node1) + int(
+            validate_sync_dir_on_mirror_node2
+        )
+        log.info(f"Total Dir synced is {total_dir_synced}")
+        if total_dir_synced == snap_count:
+            log.info("Total directory count matched snap_count")
+        else:
+            log.error("Total directory count does not match snap_count")
+            raise CommandFailed("Snapshot Synchronisation failed")
+
+        synced_snapshots = []
+        validate_synced_snaps_on_mirror_node1 = (
+            fs_mirroring_utils.extract_synced_snapshots(
+                cephfs_mirror_node[0],
+                source_fs,
+                fsid,
+                asok_file1,
+                filesystem_id,
+                peer_uuid,
+            )
+        )
+        log.info(
+            f"Total Synced directory found on {cephfs_mirror_node[0].node.hostname} is"
```

Comment:
As its not dir count, but snapshot names we printing, can the info start as "Synced directory found on ..."

Reply:
fixed